### PR TITLE
Cleanup version definition & adds filter exclusion

### PR DESCRIPTION
**Pull request by reinhapa**
- removed no longer existing schemal resource folder
- use plugins managed version
- prepare for maven resource plugin update (needs [plugin fix](https://issues.apache.org/jira/browse/MRESOURCES-301))

### CODE REVIEW

File: izpack-dist/pom.xml [hunk 1]
```diff
@@ -69,6 +65,9 @@
                             <goal>copy-resources</goal>
                         </goals>
                         <configuration>
+                            <nonFilteredFileExtensions>
+                                <nonFilteredFileExtension>jar</nonFilteredFileExtension>
```
Comment (akuhtz) [inline izpack-dist/pom.xml:69]: @reinhapa Looking at [m-r-p](https://github.com/apache/maven-resources-plugin/blob/master/src/main/java/org/apache/maven/plugins/resources/ResourcesMojo.java#L175) the parameter name is ```nonFilteredFileExtensions```.

**Comment by akuhtz** [comment]
@reinhapa Interessting that it passes under Windows:
```
[DEBUG] Copying file lib\kunststoff.jar
[DEBUG] file kunststoff.jar has a non filtered file extension
````

**Comment by reinhapa** [comment]
> The filter works when `copy-resources` is called but not when `resources` is called:

Hmm, I did not noticed that. Any idea, where this can be changed/configured? (I'm more into Gradle these days)

**Comment by akuhtz** [comment]
I don't know how to commit to a PR directly, but the patch is attached. The configuration of ```nonFilteredFileExtensions``` is moved to general configuration of the m-r-p.
[patch.zip](https://github.com/izpack/izpack/files/12669550/patch.zip)

**Review by akuhtz — APPROVED** [review]

**Pull request merged**